tekton/release-cheat-sheet.md

```diff
@@ -95,40 +98,59 @@ the pipelines repo, a terminal window and a text editor.
       create-draft-release
     ```
 
-    4. Watch logs of create-draft-release
+    1. Watch logs of create-draft-release
 
-    5. On successful completion, a URL will be logged. Visit that URL and sort the
-    release notes. **Double-check that the list of commits here matches your expectations
-    for the release.** You might need to remove incorrect commits or copy/paste commits
-    from the release branch. Refer to previous releases to confirm the expected format.
+    1. On successful completion, a URL will be logged. Visit that URL and look through the release notes.
+      1. Manually add upgrade and deprecation notices based on the generated release notes
+      1. Double-check that the list of commits here matches your expectations
+         for the release. You might need to remove incorrect commits or copy/paste commits
+         from the release branch. Refer to previous releases to confirm the expected format.
 
-    6. Publish the GitHub release once all notes are correct and in order.
+    1. Un-check the "This is a pre-release" checkbox since you're making a legit for-reals release!
 
-10. Edit `README.md` on `master` branch, add entry to docs table with latest release links.
+    1. Publish the GitHub release once all notes are correct and in order.
 
-11. Push & make PR for updated `README.md`
+1. Edit `README.md` on `master` branch, add entry to docs table with latest release links.
 
-12. **Important: Stop pointing `kubectl` at dogfooding cluster.**
+1. Push & make PR for updated `README.md`
 
-    ```bash
-    kubectl config use-context my-dev-cluster
-    ```
-
-13. Test release that you just made.
+1. Test release that you just made.
 
     ```bash
     # Test latest
-    kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+    kubectl --context dogfooding apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
     ```
 
     ```bash
     # Test backport
-    kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.11.2/release.yaml
+    kubectl --context dogfooding apply --filename https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.11.2/release.yaml
```

Comment:
@sbwsg @afrittoli is the idea that you'd run these tests directly against the dogfood cluster, or against your own cluster? (i have to admit ive always skipped this step... 😅 )

Reply:
I feel like "test" is, in hindsight, maybe too strong a word here - all I do is apply the new release to my own cluster and watch the deployments in the tekton namespace to make sure that the controller comes up successfully with the new version.

Reply:
(but yeah, sorry, to answer the question - apply them to your own cluster, not to dogfooding)

Reply:
ahhh kk, I'll change this to use the my-dev-cluster context then :D